tests/reduxTest.js

```diff
@@ -121,6 +123,34 @@ describe('handles errors', function() {
   })
 })
 
+
+describe('add to touchedfields', function () {
+  it('touches a field', function () {
+    store.dispatch(onFocus('example-form', 'blep-field'))
+    store.dispatch(onFocus('example-form', 'blep-field'))
+    store.dispatch(onFocus('example-form', 'blep-field'))
```

Comment:
Why are we dispatching onFocus to blep-field 3 times? -- make a comment that it's there to make sure it's only in the array once for future devs.

Reply:
Comment pleeeease

Reply:
Commenting!